staging/src/k8s.io/apimachinery/pkg/runtime/error_test.go

```diff
@@ -0,0 +1,22 @@
+package runtime
```

Comment:
New files need a Kubernetes APL2 copyright header with the current year

Reply:
Fixed